lib/takeaway.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'confirmation'
+require 'date'
```

Comment:
Not sure what 'date' is, please explain what file you are requiring as I couldn't find it.

Reply:
This can be deleted.